lib/pf/constants/eap_type.pm

```diff
@@ -84,7 +85,7 @@ Readonly::Scalar our $EAP_AKA2 => 50;
 Readonly::Scalar our $EAP_GPSK => 51;
 Readonly::Scalar our $EAP_PWD => 52;
 Readonly::Scalar our $EAP_EVEV1 => 53;
-
+Readonly::Scalar our $OPTION_82 => 54;
```

Comment:
Internal types should be greater than 255 to avoid a conflicts with the freeradius types.